plugins/doc_fragments/k8s_wait_options.py

```diff
@@ -64,4 +64,9 @@ class ModuleDocFragment(object):
         - The possible reasons in a condition are specific to each resource type in Kubernetes.
         - See the API documentation of the status field for a given resource to see possible choices.
     type: dict
+  wait_for:
+    description:
+    - Specifies a property on the resource to wait for.
+    - Ignored if C(wait) is not set or is set to I(False).
```

Comment:
Some more detailed syntax examples or links to syntax documentation would be helpful think